src/sentry/monitors/constants.py

```diff
@@ -12,3 +14,27 @@
 # affects the performance of recent check-ins query
 # lowering this may invalidate monitors + block check-ins
 MAX_THRESHOLD = 720
+
+
+class AcceptedCheckInStatus(Enum):
```

Comment:
No need to necessarily change since I may be thinking about this wrong and I know this is coming from Evan's other PR but `AcceptedCheckInStatus.DROP` feels a bit weird to me since it implies it's both accepted and dropped

Reply:
@mrduncan I see what you mean. What about `PermitCheckInStatus.XXX` or `AuthorizeCheckInStatus.XXX` or `ValidateCheckInStatus.XXX`?

Reply:
I like `PermitCheckInStatus` or even just `CheckInStatus` but totally your call

Reply:
`CheckInStatus` is already used by the MonitorCheckIn model's status field